app/src/main/res/layout/admin_pin_activity.xml

```diff
@@ -57,35 +56,53 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/admin_pin_main_text" />
 
-      <org.oppia.android.app.profile.ProfileInputView
+      <com.google.android.material.textfield.TextInputLayout
         android:id="@+id/input_pin"
+        style="@style/TextInputLayout"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginStart="28dp"
         android:layout_marginTop="28dp"
         android:layout_marginEnd="28dp"
-        app:inputLength="5"
-        app:isPasswordInput="true"
-        app:label="@string/admin_pin_new_pin"
+        app:errorMessage="@{viewModel.pinErrorMsg}"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/admin_pin_warning_text"
-        profile:error="@{viewModel.pinErrorMsg}" />
+        app:layout_constraintTop_toBottomOf="@+id/admin_pin_warning_text">
 
-      <org.oppia.android.app.profile.ProfileInputView
+        <com.google.android.material.textfield.TextInputEditText
+          android:id="@+id/input_pin_edit_text"
+          style="@style/TextInputEditText"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:hint="@string/admin_pin_new_pin"
+          android:imeOptions="actionDone"
```

Comment:
this should be `actionNext` so the next input gets ready for input

Reply:
Done